client/blocks/upsell-nudge/index.jsx

```diff
@@ -29,6 +29,7 @@ import isVipSite from 'calypso/state/selectors/is-vip-site';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import { getSite, isJetpackSite } from 'calypso/state/sites/selectors';
 import isSiteWPForTeams from 'calypso/state/selectors/is-site-wpforteams';
+import isSiteWpcomAtomic from 'calypso/state/selectors/is-site-wpcom-atomic';
```

Comment:
why do you use `is-site-wpcom-atomic` instead of `is-site-automated-transfer`?

Reply:
What's the differences between them? I didn't know about `is-site-automated-transfer` so that's why I used `is-site-wpcom-atomic`. It also the selector used for determining if a site can be upgraded to a given plan: https://github.com/Automattic/wp-calypso/blob/db584fece8fe1a99cb1429f44b34f78d4fe10226/client/state/selectors/can-upgrade-to-plan.js#L20

Reply:
Interestingly, `is-site-automated-transfer` is also used there 😅 

https://github.com/Automattic/wp-calypso/blob/db584fece8fe1a99cb1429f44b34f78d4fe10226/client/state/selectors/can-upgrade-to-plan.js#L19-L20

Seems that the differences between the selectors are tied to the differences between Atomic v1 and v2? See https://github.com/Automattic/wp-calypso/pull/32544.

Now I wonder which one we should use.

Reply:
> Now I wonder which one we should use.

yes, that's why I asked. I did a quick search and it seems that the automated transfer selected is more used compared with the atomic one. :-/

Reply:
According to https://github.com/Automattic/wp-calypso/pull/32544, it seems that isSiteAutomatedTransfer returns true for both Atomic v1 and Atomic v2 sites, while isSiteWpcomAtomic only returns true for Atomic v2 sites.

I guess we should use isSiteAutomatedTransfer  then. Changed in 342baab